final_project/year_app/year_app.py

```diff
@@ -1,24 +1,67 @@
-from flask import Flask, render_template
+from flask import Flask, request, session, url_for, redirect, \
+     render_template, abort, g, flash, _app_ctx_stack
 import numpy as np
 import pickle
+import requests
+import pprint
+pp = pprint.PrettyPrinter(indent=4)
+
+from pyechonest import config
```

Comment:
Nvm. I see that you do. Maybe we can just move all of our echonest api requests into the echo_req.py file?